docs/table/access_table.rst

```diff
@@ -31,7 +31,7 @@ the data contained in that object relate to the original table data
 **Table properties**
 ::
 
-  t.columns   # Dict of table columns
+  t.columns   # Dict of table columns (access by column name, number, or slice)
```

Comment:
Instead of "number" I think "index" should be used.
Other than that this looks like a good addition to the docs, thanks :+1: